Logger/LoggerServer.h

```diff
@@ -74,7 +74,7 @@ namespace Logger
 
 		private:
 			inline LoggerServer()
-			:	Network::TcpServer(defaultLoggerPort, "Logger"),
+			:	Network::TcpServer("::1", defaultLoggerPort, "Logger"),
```

Comment:
The logger TCP server is an undocumented feature that is often used on small computers with only a (lowcost) SSD stick to reduce writing activities. There it is important to use any interfaces and not only localhost.

Reply:
Railcontrol shouldn't open any undocumented network ports. Also on Linux systems it is common to use rsyslog or syslog-ng and then use theses logging facilities to redirect the output to somewhere else, i.e. Graylog. It would be a good idea to have a systemd integration for Railcontrol. But I can leave it open for now, because it hopefully not let anybody control my layout.

Reply:
RailControl is NOT reading on the logger connection. It just bumps out the log.

Reply:
Yes, that's what I understood.